.github/workflows/update-docs.yml

```diff
@@ -0,0 +1,19 @@
+name: Node.js CI
```

Comment:
I'm not sure if two workflows can have same name, could you please change it to Docs Deployment, or something?

Reply:
done